### ISSUE TITLE
nodeschool International Day May 23

### ISSUE THREAD

**Issue by afknapping** [nodeschool/hamburg#27]
![image](https://cloud.githubusercontent.com/assets/170145/7200265/9118ee28-e4fb-11e4-93df-5c16543cc12b.png)

Just got the basic ok that we could do an [international day event](http://nodeschool.io/#international) at Jimdo :)

initial thoughts
- Since weekend, let's start late at 12h. That also implies that there is no lunch at the event.
- 30 to 50 attendees (learners)
- Some snacks in the afternoon would be great
- "official" timeslot could be shorter (maybe 4h?) to lower the hurdle for mentors. Internally, we could plan to stay longer for those who want to keep hacking or talking.
- besides location, Jimdo will sponsor softdrinks (and I guess also a couple beers and proseccos to end the day)
- we should [order stickers](http://hexb.in/vector/nodeschool-int.svg)

cheers, looking forward to your feedback! :)

PS: I have not made this fix yet of course – if sb planned sth in parallel, I can easily cancel.

**Comment by finnp** [comment]
You should totally do something for international day :) I'm currently planning for Berlin as well.

@greelgorke You are printing the stickers in Hamburg, right? Maybe we could do a badge of stickers for Berlin and Hamburg. I already asked DigitalOcean for sponsoring.

**Comment by greelgorke** [comment]
@fgnass knows more about stickers printing.

Yes the international day. Would be nice to see that happen. who do want to help @nodeschool/hamburg ?

**Comment by chieftecho** [comment]
> who do want to help @nodeschool/hamburg 
> https://github.com/orgs/nodeschool/teams/hamburg ?
> 
> me!

**Comment by afknapping** [comment]
just skimmed again through https://github.com/nodeschool/hamburg/issues/11 (ticket fees).

How has it worked out at the past nodeschool hh events with no-shows? If we want to take a small ticket fee this time, it _might_ be possible to handle that through Jimdo too – at least I can ask.

**Comment by afknapping** [comment]
am working on an info page for new mentors: http://filtercake.com/nodeschool.github.io/mentor-at-event.html

not done yet, but should work already...

(issue: https://github.com/nodeschool/nodeschool.github.io/issues/286)

**Comment by afknapping** [comment]
@YildirimKaral awesome! :+1: do you want to help with orga stuff and/or do mentoring?

@fgnass would you want to take care of printing the stickers? if not, could you post info from the last sticker order at cutandprint here? (sticker specs, printing method, order nr, contact,...)

**Comment by chieftecho** [comment]
I can do both.

Am 19.04.15 um 10:52 schrieb filtercake:

> @YildirimKaral https://github.com/YildirimKaral awesome! :+1: do you 
> want to help with orga stuff and/or do mentoring?
> 
> @fgnass https://github.com/fgnass would you want to take care of 
> printing the stickers? if not, could you post info from the last 
> sticker order at cutandprint here? (sticker specs, printing method, 
> order nr, contact,...)
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/nodeschool/hamburg/issues/27#issuecomment-94252429.

## 

Viele Grüße,

   Yildirim

**Comment by fgnass** [comment]
@filtercake sorry, I don't have the time to handle this but maybe you could ask @oleharland if he could jump in and take care of the printing. He did it last time and already [joined the discussion on twitter](https://twitter.com/oleharland/status/588967260355690496).

Regarding cutandprint: We didn't use their stickers because the quality was subpar – bad colors, heavy dithering and the cutting was totally askew. We then went for http://www.frey-siebdruck.de/ to get offset prints rather than digital ones. They were quite expensive and the color started peeling off really quickly. No comparison to the matte & weatherproof stickers you get at stickermule. My recommendation would be to either go with stickermule and use http://www.shipito.com/ (or some other proxy sender) to reduce the high shipping costs or contact the guys at http://www.stickma.de/ and ask them which of their products is best suited as laptop sticker.

**Comment by greelgorke** [comment]
@filtercake with the no-shows: we had about 50% no show rate at the first event. @fgnass should know about the second. beside that, we think the free ticketing works quite ok, if you count in the 30-50% no-shows. Though the charge-and-refund model might work as well, if you are not afraid of handling money. this was the main issue for us so far: we don't want to handle money, without a legal backing behind. i'm not that comfortable with taking fee to finance the event, this would kind of shift the feeling of a community learning event in to an impression of paid learning hours, i guess.

regarding stickers: i still have a bunch of nodeschool hamburg stickers here.

PS: @filtercake you have a pending invitation to nodeschool/hamburg team. pls check your invites :)

**Comment by fgnass** [comment]
The no-show rate for the 2nd event was around 30%.

**Comment by greelgorke** [comment]
i can setup tito for the event, if needed

**Comment by al-2014** [comment]
please guys right down what's needed and deadlines and i maybe can help u in something

**Comment by al-2014** [comment]
am new to this event never participated before.

**Comment by afknapping** [comment]
thx for the feedback, all!

@fgnass: asked stickma, heard good things about them. What size are the HH stickers?

@greelgorke: +1 for tito page. also actually found that invite mail, thx for the hint! :)

@al-2014 awesome! :+1: do you want to help with orga stuff and/or do mentoring?

---

So I guess we can keep tickets free and savely overbook 30% to make up for no-shows. Will give an update when I know how many seats we can exactly provide...

In sponsoring news, I've also contacted DigitalOcean because of @finnp's hint and it seems they might sponsor food and beverages \o/ (not fixed yet, though).

**Comment by fgnass** [comment]
Here are the .ai and .eps versions of the nodeschool Hamburg stickers as reference: https://cloudup.com/cF3OoZnLYA0

According to [the specs](http://terinjokes.github.io/StickerConstructorSpec/#hexagon-sticker) the largest diagonal of the hexagon has to be 5.08cm.

**Comment by al-2014** [comment]
@filtercake tell me what is needed and sure i can help. I want to know the date of the event ?

**Comment by finnp** [comment]
I ordered 100 international day stickers from Japan. I can send you 50 of them, when they arrive.

**Comment by oleharland** [comment]
I can check if i can provide a location here in Hamburg Altona http://www.werkheim-hamburg.de/

**Comment by oleharland** [comment]
I just saw that I am to late but I can also offer a location here in altona if you want to check it out.

http://www.werkheim-hamburg.de/

Also if you need any help feel free to ask.

**Comment by ghost** [comment]
Yeah to know a date would be awesome. I'd love to participate and hhopefully finally understand node

**Comment by afknapping** [comment]
International day is Saturday May 23

**Comment by afknapping** [comment]
@oleharland 👍 would you want to do photos at the event? Re werkheim: thx for the offer. having  it at jimdo makes organizing a lot easier though, so i'd prefer to stick with it :)

@finnp thx, we might also print a batch of stickers, will let you know.

**Comment by oleharland** [comment]
@filtercake I don't know if I will attend because the Werkheim is occupied that day with the hackathon from the geekettes an I maybe attend there.

Maybe next time.

**Comment by finnp** [comment]
@filtercake That's cool in case one of the prints fails :)

**Comment by afknapping** [comment]
Some updates...

This is the space:

![image](https://cloud.githubusercontent.com/assets/170145/7429037/d23a83b2-effa-11e4-8608-6c4e6f9b42a7.png)
- fridge with softdrinks ✔️
- beamer ✔️
- we can offer 50 seats for attendees. overbooking 30% because noshows would result in 65 attendee tickets
- while our guest wifi usually works, I've started to talk to folks if we can start using [Freifunk](https://hamburg.freifunk.net/worum-gehts).
- we have a Kita at Jimdo, so there would be space for child care. I'll get feedback next week if we can provide that via the Jimdo day nannies. If not I'll try to set it up otherwise
- we decided to move the event forward a bit [because soccer](http://www.weltfussball.de/spielplan/bundesliga-2014-2015/), so the new kernnodingzeit for attendees is 11:00 to 15:00.
- there will be soccer watching on the beamer from 15:30
- RFC: maybe we put 10:30 as "official starting time"? always nice to have some buffer time for setup, chat & coffee and people coming late...

I will open an issue where mentors can RSVP. iirc the recommended ratio mentors : attendees is 1:5? So it would be nice to have 10 mentors at the event. To make up for cancelations, I'd say the goal is to have 15 on the list.

**Comment by greelgorke** [comment]
@filtercake 1:5 to 1:10 is acceptable. depends on the public though. 

do you still need the tito page?

**Comment by afknapping** [comment]
@greelgorke "you"? you're not taking part? :)  however, yes please set it up and post the link, i'll update the next event section on nodeschool/hamburg then...

**Comment by ghost** [comment]
Do I need to get somehow a ticket for this event? How can I get my hands on one? I'd love to come and learn node.

**Comment by afknapping** [comment]
@sebastian301082 awesome! :) we'll have the ticket link up soon...

**Comment by greelgorke** [comment]
@filtercake i wont be able to participate, sorry about that.

however here is the tito page:

https://ti.to/nodeschool-hamburg/international-day

i need your mail address to add you to tito team, so you can change the details and access the attenders list

**Comment by afknapping** [comment]
@greelgorke thx! `fabian at superpursuitmo.de`

**Comment by greelgorke** [comment]
@filtercake tap you're in

**Comment by YildirimKaral** [comment]
thanks @greelgorke <https://github.com/greelgorke> ! |yildirim at karal 
dot eu|

Am 11.05.15 um 14:21 schrieb Gregor Elke:
>
> @filtercake <https://github.com/filtercake> i wont be able to 
> participate, sorry about that.
>
> however here is the tito page:
>
> https://ti.to/nodeschool-hamburg/international-day
>
> i need your mail address to add you to tito team, so you can change 
> the details and access the attenders list
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/nodeschool/hamburg/issues/27#issuecomment-100890022>.
>

-- 
Viele Grüße,

   Yildirim

**Comment by afknapping** [comment]
@greelgorke thx!

and one last update: we will not do childcare this time :/ lots of unclear questions about insurance and other legal stuff. but: it got us started to work on it and we have commitment to offer that for upcoming events, so hopefully next time.

**Comment by afknapping** [comment]
[draft for the ticket page. I had people ask me about the laptop thing, so I included it...]

---

**Die NodeSchool eine Veranstaltung um mit anderen Interessierten gemeinsam Node.js und JavaScript zu lernen - ganz einfach mit kleinen "Programmier-Spielen".**

**Learn Node.js and JavaScript together with interactive open source courses and helping mentors.**

+++ Just in case: You _need_ to bring your own laptop! :) +++

## Timetable

10:30 open doors
11:00 welcome and introduction of mentors
11:15 workshop start
15:00 "official end" of workshop time
15:30 Bundesliga on the beamer

There will be softdrinks and some snacks available during the event.

Read more / mehr unter: http://nodeschool.io/hamburg/

**Comment by afknapping** [comment]
Ok, please help to get the word out everyone :)

Here's a copy-paste ready tweet template:

```
Want to learn node.js and JavaScript?

Join us at the @nodeschool Int’l Day Hamburg, May 23.

Tickets are free at: https://ti.to/nodeschool-hamburg/international-day
```

**Comment by afknapping** [comment]
Tickets are going away a bit slow, so I'm going to post the following at several usergroups today...

---

Hey everyone!

Just wanted to let you know that there is a NodeSchool event coming up this Saturday, May 23. If you ever wanted to look into Node.js, grab a free ticket at (url).

Have a great week, hope to see you on Saturday!

**Comment by ghost** [comment]
How much of the event will be in English? Some friends doesn't speak German but consider to come as well.

**Comment by afknapping** [comment]
@sebastian301082 usually at these kinds of events, we ask at the beginning if anyone prefers english and if one person does, well, then the speaking language is english. we also usually prepare any handouts or slides in english for that case. so, yes, please ping them to attend :)

**Comment by ghost** [comment]
@filtercake Thank you!!!

**Comment by sas05** [comment]
I wish workshop will be in English

**Comment by chieftecho** [comment]
Hi all,
I'm very sorry that I can't participate today, but we have an emergency in the family, and I have to take care of the kids!

Wish you a lot of fun.

Best, Yildirim

**Comment by bruderstein** [comment]
I'm going to be late. Should be there about 11:30. Ersatzverkehr, Ersatzverkehr, überall Ersatzverkehr!

**Comment by ghost** [comment]
Big thank you to all the mentors and organizers. This was an really helpful event. Keep on doing events like this.

THANK YOU!!!!!

**Comment by afknapping** [comment]
@sebastian301082 thx Sebastian :)

**Comment by afknapping** [comment]
So that went well except for having very few attendees (probably mostly due to Pfingstwochenende). We're shooting to have another one at Jimdo on Sat Oct 10.

---

![image](https://cloud.githubusercontent.com/assets/170145/7877292/24e223ba-05d9-11e5-81c0-159075cd8632.png)

![image](https://cloud.githubusercontent.com/assets/170145/7877294/2b2d70d0-05d9-11e5-92ed-fcec1370614e.png)

**Comment by greelgorke** [comment]
Nice !
